integrated_channels/xapi/utils.py

```diff
@@ -26,25 +26,19 @@ def send_course_enrollment_statement(lrs_configuration, course_enrollment):
          lrs_configuration (XAPILRSConfiguration): XAPILRSConfiguration instance where to send statements.
          course_enrollment (CourseEnrollment): Course enrollment object.
     """
-    user_details = LearnerInfoSerializer(
-        course_enrollment.user,
-        context={'enterprise_customer': lrs_configuration.enterprise_customer}
-    )
-    course_details = CourseInfoSerializer(
-        course_enrollment.course,
-        context={'site': lrs_configuration.enterprise_customer.site}
-    )
+    user = course_enrollment.user
     LOGGER.info(
         'Sending course enrollment to xAPI for user: {username} for course: {course_key}'.format(
```

Comment:
Tangential -- do we capture if/when the send fails?

Reply:
we are capturing it [here](https://github.com/edx/edx-enterprise/blob/877511923e5b2a309bc8cb7a374cdace1fee7073/integrated_channels/xapi/management/commands/send_course_enrollments.py#L127)